smrf/distribute/precipitation.py

```diff
@@ -324,6 +342,13 @@ def distribute(self, data, dpt, precip_temp, ta, time, wind, temp,
                     self.veg,
                     self.config)
 
+        # scale the post distribution data based on user config
+        output_scalar_key = get_scalar_value_key(self.config, "output")
+        if output_scalar_key is not None:
```

Comment:
I like this idea a lot. Are you suggesting the `output_scalar_key` be the property, or we make a property for something like `scaled_data`?

Or should I make this a property on the `Modifiers` class that @jomey suggested a couple comments down?

Reply:
I was thinking more of the `scaled_data` as a property. So it could even live on `image_data` at some point if this is applied to all the variables.

With it being on `Modifiers`, then you'd have to initiate a class and call the method anyways. Wouldn't that have to still live in the `@property` regardless?

Reply:
The latests changes has a `Modifiers` class that is a `@property` in the precip class. It would be easy to transfer to the `image_data` class when we're ready for that.